setup.py

```diff
@@ -6,7 +6,8 @@
 
 setup(
     install_requires=[
-        "requests"
+        "requests",
+        "camel_converter"
```

Comment:
I believe pypi will understand this and convert it but the package name is `camel-converter` on pypi so if there are still issues that is probably it.

Reply:
Do you think we should release again with `camel-converter` instead?

Reply:
I tested `pip install camel_converter` and pypi did understand and install correctly so I think it is fine.

Reply:
I tested and it did install, but there is still an issue:

```console
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/paul/t/.venv/lib/python3.10/site-packages/meilisearch/__init__.py", line 1, in <module>
    from meilisearch.client import Client as Client  # pylint: disable=useless-import-alias
  File "/home/paul/t/.venv/lib/python3.10/site-packages/meilisearch/client.py", line 9, in <module>
    from meilisearch.index import Index
  File "/home/paul/t/.venv/lib/python3.10/site-packages/meilisearch/index.py", line 11, in <module>
    from meilisearch.models.task import Task, TaskInfo, TaskResults
  File "/home/paul/t/.venv/lib/python3.10/site-packages/meilisearch/models/task.py", line 2, in <module>
    from camel_converter.pydantic_base import CamelBase
  File "/home/paul/t/.venv/lib/python3.10/site-packages/camel_converter/pydantic_base.py", line 4, in <module>
    raise ImportError("camel-converter must be installed with the pydantic extra to use this class")
ImportError: camel-converter must be installed with the pydantic extra to use this class
```

It needs to be `camel-converter[pydantic]` to include the extras

Reply:
Thank you @sanders41 I will try to fix it today!